packages/block-editor/src/layouts/flow.js

```diff
@@ -174,32 +179,40 @@ export default {
 			alignments.unshift( { name: 'full' } );
 		}
 
-		/**
-		 * Besides checking if `contentSize` and `wideSize` have a
-		 * value, we now show this information only if their values
-		 * are not a `css var`. This needs to change when parsing
-		 * css variables land.
-		 *
-		 * @see https://github.com/WordPress/gutenberg/pull/34710#issuecomment-918000752
-		 */
 		if ( wideSize ) {
-			const wideAlignment = { name: 'wide' };
-			if ( ! wideSize?.startsWith( 'var' ) ) {
-				// translators: %s: container size (i.e. 600px etc)
-				wideAlignment.info = sprintf( __( 'Max %s wide' ), wideSize );
-			}
-			alignments.unshift( wideAlignment );
+			alignments.unshift( { name: 'wide', info: alignmentInfo.wide } );
 		}
 
-		// Add `none` alignment with info text.
-		if ( contentSize && ! contentSize?.startsWith( 'var' ) ) {
-			alignments.unshift( {
-				name: 'none',
-				// translators: %s: container size (i.e. 600px etc)
-				info: sprintf( __( 'Max %s wide' ), contentSize ),
-			} );
-		}
+		alignments.unshift( { name: 'none', info: alignmentInfo.none } );
 
 		return alignments;
 	},
 };
+
+/**
+ * Helper method to assign contextual info to clarify
+ * alignment settings.
+ *
+ * Besides checking if `contentSize` and `wideSize` have a
+ * value, we now show this information only if their values
+ * are not a `css var`. This needs to change when parsing
+ * css variables land.
+ *
+ * @see https://github.com/WordPress/gutenberg/pull/34710#issuecomment-918000752
+ *
+ * @param {Object} layout The layout object.
+ * @return {Object} An object with contextual info per alignment.
+ */
+function getAlignmentsInfo( layout ) {
+	const { contentSize, wideSize } = layout;
+	const alignmentInfo = {};
+	if ( contentSize && ! contentSize?.startsWith( 'var' ) ) {
```

Comment:
What about `calc` and other functions? It feels instead of checking for specific function we should just check that the value is a numeric followed by an optional unit using a regex.

Reply:
You're right! I'll change it.